applications/omnipresence/src/omnip_subscriptions.erl

```diff
@@ -407,6 +407,7 @@ subscribe_to_record(JObj) ->
                         ,subscription_id=wh_json:get_value(<<"Subscription-ID">>, JObj)
                         ,proxy_route= wh_json:get_value(<<"Proxy-Route">>, JObj)
                         ,version=Version
+                        ,user_agent=wh_json:get_value(<<"User-Agent">>, JObj)
```

Comment:
Sorry I never really understood it: when should we call the `get_ne_binary_value` or equivalent, given that we also have just `get_value`?